src/main/kotlin/io/spotfire/solver/domain/SolverHandlerPayload.kt

```diff
@@ -0,0 +1,12 @@
+package io.spotfire.solver.domain
+
+import kotlinx.serialization.Serializable
+@Serializable
+data class Customer(val id: String, val firstName: String, val lastName: String, val email: String)
```

Comment:
was this meant to be an example payload?

Reply:
Yeah, i need to remove that, it was me taking notes and didn't notice i left it behind